config/defaults.go

```diff
@@ -158,3 +160,14 @@ func getServerDefaults() *ServerConfig {
 		Port:         3003,
 	}
 }
+
+func generateRandomDeploymentIDSufix() string {
```

Comment:
maybe it's easier to just use google/uuid package and generate uuid v4?
Especially as this package is already in indirect dependencies.

Reply:
done